src/Tribe/Aggregator/Cron.php

```diff
@@ -385,6 +386,75 @@ public function verify_child_record_creation() {
 		}
 	}
 
+	public function start_batch_pushing_records() {
+		if ( ! tribe( 'events-aggregator.main' )->is_service_active() ) {
+			return;
+		}
+
+		$records = Tribe__Events__Aggregator__Records::instance();
+
+		$query = $records->query( [
+			'post_status'    => Tribe__Events__Aggregator__Records::$status->pending,
+			'posts_per_page' => 15,
+			'order'          => 'ASC',
+			'meta_query'     => [
```

Comment:
:warning: Detected usage of meta_query, possible slow query.

<sub>hash: f888f7c0913874ee1868ebec63fccc18</sub>